grid/Grid-Recon.md

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Add a  header could you add:

```
Last reviewed: Feb 2021
Reviewed by: Baye
```

so we have a clear log for finding out-of-date research

Reply:
Updated